generators/fusesoc

```diff
@@ -29,11 +29,26 @@ except IndexError:
     print("Usage: ./generator <subdir>")
     sys.exit(1)
 
+try:
+    runner = os.environ['RUNNERS_FILTER']
+except KeyError:
+    runner = None
+
+tests_list_exclude = []
+
+if runner in ["UhdmYosys"]:
```

Comment:
This needs to be moved to the generic runner code (as discussed offline).

The runners need to check whether a specific test is intended for them and then skip it if it's not. This implementation will not work

Reply:
Done, I added new meta-tag: ``compatible-runners`` to check whether a specific test is for given runner.